File: src/main/java/ru/ioleksiv/telegram/bot/api/model/annotation/validator/impl/MaskPositionFilterValidator.java
package ru.ioleksiv.telegram.bot.api.model.annotation.validator.impl;

import ru.ioleksiv.telegram.bot.api.model.annotation.validator.CustomValidator;
import ru.ioleksiv.telegram.bot.api.model.annotation.validator.FilterValidator;
import ru.ioleksiv.telegram.bot.api.model.objects.std.sticker.MaskPosition;

public class MaskPositionFilterValidator extends FilterValidator<MaskPosition> {

    public MaskPositionFilterValidator(CustomValidator<MaskPosition> validator) {
        super(MaskPosition.class, validator);
    }
}

File: core/src/main/kotlin/io/github/oleksivio/tl/kbot/core/model/annotation/validator/impl/MaskPositionFilterValidator.kt
package io.github.oleksivio.tl.kbot.core.model.annotation.validator.impl

import io.github.oleksivio.tl.kbot.core.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.std.sticker.MaskPosition
import io.github.oleksivio.tl.kbot.core.core.controller.handler.check.Validator

class MaskPositionFilterValidator(validator: Validator<MaskPosition>) : FilterValidator<MaskPosition>(
    MaskPosition::class, validator)
